### PR TITLE
A3qz/balance/enemy scaling

### PR DESCRIPTION
**Pull request by a3qz**

### CODE REVIEW

File: item.c [hunk 1]
```diff
@@ -138,7 +137,10 @@ void item_drink(item_t* item) {
                 } else {
                     add_action("It did nothing.");
                 }
-                get_player_obj()->luck += item->power*2;
+                get_player_obj()->luck += item->power*100;
+				if(get_player_obj()->luck < 2){
+					get_player_obj()->luck = 2;
```
Comment (JohnathonNow) [inline item.c]: why is this tabbed

File: key.c [hunk 2]
```diff
@@ -44,7 +44,7 @@ void key_add_stair(int dir, int pic){
  }
 
  void key_checker(WINDOW *win, int y, int x){
-	int starting_number = 10;
+	int starting_number = 11;
```
Comment (JohnathonNow) [inline key.c]: hmmmm

Comment (a3qz) [reply key.c]: this fixes the key

File: item.c (hunk 1 above)
Comment (a3qz) [reply item.c]: for ricing

**Pull request merged**